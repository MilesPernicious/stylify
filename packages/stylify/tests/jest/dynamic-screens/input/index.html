<div class="
	color:red
	minw640px:color:blue
	minw1024px&&maxw1280:color:green
	minw1280px||landscape:color:purple
	minw1280px||landscape:hover:color:purple
">
</div>
