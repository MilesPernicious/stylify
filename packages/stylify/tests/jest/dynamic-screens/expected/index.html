<div class="
	s0
	s1
	s2
	s3
	s4
">
</div>
